incubator/solr/values.yaml

```
@@ -0,0 +1,81 @@
+replicaCount: 3
+
+revisionHistoryLimit: 5
+
+updateStrategy:
+  type: RollingUpdate
+
+podManagementPolicy: "OrderedReady"
+
+zkClientTimeout: 15000
+
+terminationGracePeriodSeconds: 1800
+
+ports:
+  client:
+    containerPort: 8983
+
+livenessProbe:
+  # tcpSocket:
+    # port: 8983
```

I would recommend using port names here and other such instances.

```suggestion
    # port: solr-port
```

Also, liveness and readiness probes should be enabled by default.